doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -469,10 +456,19 @@ arguments used for configuration.  Here is a table describing these emulated dev
 
    * - ``virtio-net``
      - Virtio network type device, parameter should be appended with the format:
-       ``virtio-net,<device_name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>]``.
-       The ``mac`` address is optional, ``device_name`` is the name of the TAP (or MacVTap) device.
-       It must include the keyword ``tap``.
+       ``virtio-net,<device_type=name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>],[mac_seed=<seed_string>]``.
+       The ``mac`` address is optional, ``name`` is the name of the TAP (or MacVTap) device.
```

Comment:
Can you also add the various values that `device_type` can take?

Reply:
Update:
"device_type" is the type of the TAP (or MacVTap) device. It must include the keyword ``tap`` .

Reply:
That's not what we had discussed, there should not be any assumption about the name.

Reply:
We should redesign the device name or the type and name logic?

Reply:
Take a look at #6727 as well .

Reply:
change to tap=tap0